server/server/Routes/admin.js

```diff
@@ -7,6 +7,10 @@ router.post("/add", adminController.addNewItem);
 router.patch("/update/:id", adminController.updateItem);
 router.delete('/remove/:ids', adminController.removeOneOrManyItems);
 router.get('/filter', adminController.getItemById);
+router.post("/add", adminController.addnewRestaurant);
+router.patch("/update/:id", adminController.updateRestaurant);
+router.delete('/remove/:ids', adminController.removeOneOrManyRestaurants);
+router.get('/filter', adminController.getRestaurantById);
```

Comment:
Item and restaurant CRUD routes should be in separate files, but it's ok for now